addons/tasks.json

```diff
@@ -4,7 +4,7 @@
       {
         "label": "Start Home Assistant",
         "type": "shell",
-        "command": "supervisor_run",
+        "command": "sudo chmod a+x /usr/bin/supervisor* && sudo -E supervisor_run",
```

Comment:
We should change permissions on the fly. Instead, lets ensure the source has the right permissions.

Reply:
Do you mean "we shouldn't change permissions on the fly"?

I have a minimal understanding of how the multiple docker containers and sources work, so if there is a better spot to put a fix to solve this problem, that sounds great to me. This did feel a bit hacky but it did get my project moving forward at least.

Reply:
I mean the permissions of `supervisor_run` should be adjusted in the source repository and not here.

Reply:
I'm looking at these things now, but very out of my element with bash, linux, and docker all together here. What you makes sense that it should be in the source.

I'm happy to help with that if I can; should the fix be in the code for `supervisor_run` in this repo here? That doesn't quite make sense to me, since the issue is that the user doesn't have permission to run the code at all, not that the code doesn't work once run.

That makes me think the fix would need to be in the devcontainer/image itself, which I can't figure out how to even find where that code would be changed.

Or a change could be in the `devcontainer.json` I'm sure, but that seems just as hacky as where I put it in this PR.

I also don't mind if it is a 2-minute fix for you or someone else for someone else to do it.